GUI/MainWindow.xaml

```diff
@@ -391,6 +391,9 @@
                         
                         in .xml or .fasta format. Reference proteomes in .xml format can contain post-translational modification (PTM) locations,
                         whereas .fasta format only contains amino acid sequences.
+                        <LineBreak />
+                        Add spectral libraries in .msp format.
+                        <LineBreak />
```

Comment:
I'd like to suggest a minor edit here, swapping out lines 394-396 with the following:

```
<Hyperlink NavigateUri="https://chemdata.nist.gov/dokuwiki/doku.php?id=peptidew:start" RequestNavigate="Hyperlink_RequestNavigate">
     Spectral libraries
</Hyperlink>

in .msp format can be added to improve search results.
```

Reply:
Thanks. It will be addressed in the new pull request.